gbBuild.py

```diff
@@ -342,164 +297,394 @@
-            #To the surprise of hopefully noone, our products are not suitable for applications which require
-            #sub-.1 meter accuracy (true limits will be much higher than this, due to data accuracy).
-            write = ("mapshaper-xl 6gb " + workingPath + row["boundaryID"] + ".geoJSON" +
-                    " -clean gap-fill-area=500m2 snap-interval=.00001" +
-                    " -o format=shapefile " + shpOUT +
-                    " -o format=topojson " + topoOUT +
-                    " -o format=geojson " + jsonOUT)
-            
-            os.system(write)    
-
-            #Do a second write, this time with simplification.
-            #Simplification attempts to keep around 100-meter resolution along boundaries.
-            write = ("mapshaper-xl 6gb " + workingPath + row["boundaryID"] + ".geoJSON" +
-                    " -simplify dp interval=100 keep-shapes" +
-                    " -clean gap-fill-area=500m2 snap-interval=.00001" +
-                    " -o format=shapefile " + shpOUT_simp +
-                    " -o format=topojson " + topoOUT_simp +
-                    " -o format=geojson " + jsonOUT_simp)
-
-            os.system(write)  
-
-            dta.boundary.plot(edgecolor="black")
-            if(len(row["boundaryCanonical"]) > 1):
-                plt.title("geoBoundaries.org - " + buildType + "\n" + row["boundaryISO"] + " " + row["boundaryType"].upper().replace(" ","") + "(" + row["boundaryCanonical"] +")" + "\nLast Source Data Update: " + str(row["sourceDataUpdateDate"]) + "\nSource: " + str(row["boundarySource-1"]))
-            else:
-                plt.title("geoBoundaries.org - " + buildType + "\n" + row["boundaryISO"] + " " + row["boundaryType"].upper().replace(" ","") + "\nLast Source Data Update: " + str(row["sourceDataUpdateDate"]) + "\nSource: " + str(row["boundarySource-1"]))
-            plt.savefig(imgOUT)
-
-            shutil.make_archive(workingPath + row["boundaryID"], 'zip', basePath)
-            shutil.move(workingPath + row["boundaryID"] + ".zip", fullZip)
-                
-
-        csvR.append(row)
-
-# Saved CSV as an artifact - TBD if this code stays here, or just log.
-
-try:
-    keys = csvR[0].keys()
-    with open(os.path.expanduser("~") + "/artifacts/results"+str(buildType)+".csv", "w") as f:
-        writer = csv.DictWriter(f, keys)
-        writer.writeheader()
-        writer.writerows(csvR)
-except:
-    print("No CSV log to output.")
-
-try:
-    #Copy the log over for an artifact
-    os.system("mv " + os.path.expanduser("~") + "/tmp/" + str(buildType) + ".txt" +" " + os.path.expanduser("~") + "/artifacts/log"+str(buildType)+".txt")
-except:
-    print("No log to output.")
-
-if(row["META_requiredChecksPassing"] != True or row["GEOM_requiredChecksPassing"] != True):
-    print("At least one check failed.  Stopping build.")
-    sys.exit("Either a metadata or Geometry check failed.  Exiting build.")
```

Comment:
this is only referencing the last row that was run. I think I'll just remove this? what was the intent here

Reply:
I think the concern might already be covered by this code:

```
        except:
            if args.buildVer == "nightly":
                row["status"] = "FAIL"
            else:
                print(
                    "The meta.txt file was not parsed correctly for at least one file.  To make a release build, all checks must pass.  Try running a nightly build first. Exiting."
                )
                sys.exit(1)
```

located earlier in the script